### ISSUE TITLE
Only Highlight Selection + Toggle Key

### ISSUE THREAD

**Issue by david1309** [jcb91/jupyter_highlight_selected_word#14]
Hello,
Thank you very much for this extension.

I was trying to configure the extension in order for it to only highlight when the **whole** word is actually selected, rather than highlighting the words when I simply put the cursor between or beside a word (which is whats happening right now). How could I achieve the former configuration ?

Moreover, **only** want to highligh  when I press `ctrl` ... how should I configure it ?

This is the configuration code I executed in python but it does not achieve what I mentioned above:

```python
from notebook.services.config import ConfigManager
cm = ConfigManager()
cm.update('notebook', {'highlight_selected_word': {
    'delay': 1000,
    'code_cells_only': True,
    'highlight_color': '#72055b',
    'highlight_color_blurred': '#72055b',
    'words_only': True,    
    'use_toggle_hotkey': True,
    'toggle_hotkey': 'ctrl',
}})
```

**Comment by hiiwave** [comment]
Hello, for the first question, you could set `show_token = ''` (see #10 for more information)

Have no idea about the second one.

**Comment by jcb91** [comment]
> Moreover, only want to highligh when I press ctrl ... how should I configure it ?

With the current version, I don't think this is possible. You could set the hotkey to *toggle* highlighting on/off (as you seem to have attempted to do), though this will require one keypress for on and another for off. However even then, I'm not aware of a way to set a hotkey to just `ctrl` without another key for it to modify (as in `ctrl-h` or similar), because the keyboard manager ignores events which only contain modifier keys (`alt`, `ctrl`, `shift`, `super`). I guess it might be *possible* to bind a custom handler to a modifier only, independently of the Jupyter keyboard manager, but I'm not sure how well that might interact with anything else going on. Does that make sense?

**Comment by jcb91** [comment]
also, thanks @hiiwave for speedy advice :smile:

**Comment by david1309** [comment]
Thank you very much @hiiwave , what you mentioned did work with the following settings:

```python
'show_token': '',
'highlight_only_whole_words': True,
```

Regarding the usage of the toggle key I really do not understand what this does @jcb91 . I did the following configuration but pressing `ctrl-i` had no effect ... what is the toggle key supposed to do, I see no effect ?

```python
'toggle_hotkey': 'ctrl-i',
'use_toggle_hotkey': True,
```

**Comment by jcb91** [comment]
@david1309 my apologies, it seems I never finished the hotkey properly :open_mouth: :expressionless:. The hotkey is intended to toggle the highlighting behaviour on or off. However, although 9b9c5d667e29300f2c4b08923c069d16041d0f65 introduced the action, I failed to actually bind the action to the hotkey. I've pushed a fix as PR #15, if you'd like to try that?

**Comment by david1309** [comment]
hahaha no worries @jcb91 . Since you've already pushed the fix, should I reinstall the extension ?

```Python
pip uninstall jupyter_highlight_selected_word
pip install jupyter_highlight_selected_word
jupyter nbextension enable highlight_selected_word/main
```

**Comment by jcb91** [comment]
I've pushed the fix to the repo master, but it won't be available from pypi yet, so you'd need to modify the second line to

    pip install jupyter_highlight_selected_word

**Comment by jcb91** [comment]
Ugh, sorry, to 

    pip install https://github.Com/jcb91/jupyter_highlight_selected_word/archive/master.zip

**Comment by david1309** [comment]
Thanks a lot for your help :D

**Comment by jcb91** [comment]
the fix is included in release 0.0.11, now available on pypi, and shortly to appear on the conda-forge channel.
